lib/active_model_serializers/adapter/json_api/meta.rb

```diff
@@ -18,7 +18,7 @@ def initialize(serializer)
 
           # Use the return value of the block unless it is nil.
           if serializer._meta.respond_to?(:call)
-            @value = instance_eval(&serializer._meta)
```

Comment:
lib/active_model/serializer/field.rb.

Attribute, and Reflection inherit from it.